packages/angular/tsconfig.ngc.json

```diff
@@ -0,0 +1,26 @@
+/* To learn more about this file see: https://angular.io/config/tsconfig. */
```

Comment:
This is helpful. Could you add similar links to the other new json files?

Reply:
Good itdea! I added it to `angular.json` and it worked fin. Then I also wanted to add it to `ng-package.json` which gave me a build error. Apparently, `ng-packagr` can't handle JSON with comments.

Reply:
Oh, yeah, `package.json` files can't handle comments (and the npm people have said they never will), so I guess it's not super surprising that `ng-package.json` can't, either.